src/ShopBundle/objects/TCMSSmartURLHandler/TCMSSmartURLHandler_ShopPaymentOgone.class.php

```diff
@@ -23,7 +23,10 @@ class TCMSSmartURLHandler_ShopPaymentOgone extends TCMSSmartURLHandler_ShopBaske
      *
      * If response from Ogone contains "ogone_cms_call" the trigger der parent function to get pagedefid.
      *
-     * @return bool
+     * @return bool|string
+     *
+     * @psalm-suppress UndefinedPropertyAssignment
+     * @FIXME Writing data into `$OURLData` when there is no magic `__set` method for them defined.
```

Comment:
In this class, there is a lot of writing into 'magic' properties of `TCMSSmartURLData` - however, those properties are not writable.